kdc/fast.c

```diff
@@ -373,8 +373,8 @@ _kdc_fast_mk_error(astgs_request_t r,
 			    NULL,
 			    error_client,
 			    error_server,
-			    NULL,
-			    NULL,
+			    csec,
```

Comment:
Did we have a bug here?

Reply:
Possibly? Starting with commit 9b55215a2aece1268c9c2cf84ebc8e82cb72625a (“kdc: sync KDC FAST with Heimdal-597.121.1”) we’ve omitted `csec` and `cusec` completely when FAST is in use. Including them in the inner error is more consistent with the handling of `r->e_text`, `error_client`, and `error_server`, which are included in the inner error but omitted from the outer.